### ISSUE TITLE
Defibrillator icon is too small

### ISSUE THREAD

**Issue by TheAdventurer64** [organicmaps/organicmaps#3306]
![image](https://user-images.githubusercontent.com/50059322/187641320-daa8b26d-b58d-44cb-ba3b-263263c8ea28.jpeg)

**Comment by JLZIMMERMANN** [comment]
In a general case I think All the pictograms need a bigger size.
In my idea and the last levels of zoom all icons should take more and more place for people that do not see very well.

**Comment by biodranik** [comment]
There is an option in settings to increase fonts. It can increase icons as well. But for that, we need to support the rendering of SVG icons (to refactor our rendering engine).

**Comment by JLZIMMERMANN** [comment]
Yes I activated it and according to me the icons displaying is very small.
